### PR TITLE
fix: /matters/author pubDate missing

### LINKED ISSUE #7227: Matters.news的pubDate缺失
#### 路由地址（不包含参数）
/matters/
```routes
```

#### 完整路由地址，包含所有必选与可选参数
/matters/author/Luterngun
```fullroutes
```

#### 相关文档地址

- https://docs.rsshub.app/...

#### 预期是什么
item中應包含pubDate 
#### 实际发生了什么
item中不包含pubDate
#### 部署相关信息

| Env                | Value         |
| ------------------ | ------------- |
| OS                 |               |
| Node version       |               |
| if Docker, version |               |

- 额外信息（日志、报错等）

### PR DESCRIPTION
**Pull request by notofoe**
Co-authored-by: @NodeBE4 

## 该 PR 相关 Issue / Involved issue

Close #7227

## 完整路由地址 / Example for the proposed route(s)

```routes
/matters/author/Luterngun
```

## 新RSS检查列表 / New RSS Script Checklist

- [ ] 这个PR中包含了新的路由吗? Does this PR add new route?
  - 如果有, 请完成检查列表. If yes, please finish the check list
  - **如果你的PR符合下方某个事项, 也请注明. If any of the checklist item meets your PR, please fill it out.**
  - [x] <- 这样打勾
- [ ] 是否提供了文档? Documentation provided?
  - [ ] 是否提供了英文文档? EN Documentation provided?
- [ ] 是否支持全文获取? Is this RSS Script support fulltext?
  - [ ] 如果全文获取中需要访问文章链接, 是否使用了缓存? If fulltext requires to fetch detail pages, is cache used in the process?
  - [缓存说明](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun) | [How to use cache](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun)
- [ ] 目标是否有明显的反爬/频率限制? Is there any sign of anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? (延长缓存时间, 写文档说明, etc.) If yes, do your code reflect this sign? (e.g. write documentations, use long cache time)
- [x] 目标是否有提供日期？ Is there a date in the source?
  - [x] 如果有，包是否正确解析？ If there is, can this script provide this info?
  - [x] 如果有提供解析能力，时区是否正确调整？ Is the timezone correctly provided?
  - 如果有提供日期，但是没有提供解析，请说明原因 If there is a date but this script does not parse, please provide your reason.
- [ ] 是否引入的新的包? Any new package introduced?
  - 如果有, 请说明原因. If yes, please state your reason
- [ ] 是否使用了`Puppeteer`? Make use of `Puppeteer`?
  - 如果有, 请说明原因. If yes, please state your reason

### CODE REVIEW

File: lib/routes/matters/author.js [hunk 1]
```diff
@@ -40,6 +42,7 @@ module.exports = async (ctx) => {
             title: article.title,
             author: user.displayName,
             description: article.content,
+            pubDate: parseDate('article.createdAt'),
```
Comment (t829702) [inline lib/routes/matters/author.js:45]: the date is always in ISO format, why nott justt call new Date?

```js
> new Date("2021-07-24T06:30:31.456Z")
2021-07-24T06:30:31.456Z
```

**Comment by t829702** [comment]
https://rsshub-do-not-use-723lwjizj-diy.vercel.app/matters/latest/heat is ok,

https://rsshub.app/matters/latest/heat but why the production deployment is failing?

```
Looks like something went wrong
Route requested: /matters/latest/heat
Error message: Timeout awaiting 'request' for 30000ms
```

**Comment by SettingDust** [comment]
> may not be relevant to changes in this PR:
> 
> https://rsshub-do-not-use-723lwjizj-diy.vercel.app/matters/latest/heat is ok,
> 
> https://rsshub.app/matters/latest/heat but why the production deployment is failing?
> 
> ```
> Looks like something went wrong
> Route requested: /matters/latest/heat
> Error message: Timeout awaiting 'request' for 30000ms
> ```

Don't spam. Open ticket in issue

File: lib/routes/matters/author.js (hunk 1 above)
Comment (SettingDust) [reply lib/routes/matters/author.js:45]: It's standard

**Comment by notofoe** [comment]
Deprecated.

**Pull request closed without merging**